automated/linux/armnn/armnn-unit-tests.yaml

```diff
@@ -0,0 +1,23 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: armnn unit tests
+    description: "Carry out armnn unit tests."
+    maintainer:
+        - theodore.grey@linaro.org
+    os:
+        - debian
+    scope:
+        - functional
+    devices:
+        - synquacer
+        - dragonboard-845c
+        - hi960-hikey
+        - stm32mp1
+params:
+    LINK_SNAPSHOT_64BIT: 'https://snapshots.linaro.org/componenets/armnn/latest/armnn.tar.xz'
+    LINK_SNAPSHOT_32BIT: 'https://snapshots.linaro.org/componenets/armnn-32bit/latest/armnn-32.tar.xz'
+    SKIP_INSTALL: false
+run:
+    steps:
+        - cd ./automated/linux/armnn/
+        - ./armnn-unit-tests.sh -s "${SKIP_INSTALL}" -a "${LINK_SNAPSHOT_64BIT}" -b "${LINK_SNAPSHOT_32BIT}"
```

Comment:
Then we can only have one of the flags...
What happens if I use a rootfs that already have my armnn preinstalled? then I want to  be able to run the test too... but not build and stuff...